### PR TITLE
Makes Port-a-Medbay send PDA notifications

### PR DESCRIPTION
**Pull request by BatElite**
[feat]
## About the PR 
This PR makes the Port-a-Medbay send out PDA messages to the medbay and med research mailing groups if it's sent home with someone inside. There's three types of messages: one for corpses, one for patients in crit (pictured) and one for general patients.

PaM-preview

## Why's this needed? 
Alerts doctors to emergencies and bodies coming in automatically. People using it in-game have been resorting to yelling at medbay over the radio to check the port-a-med after they send it, because it'd otherwise often go unnoticed.

## Changelog

```
(u)BatElite:
(*)The Port-a-Medbay now sends notifications to medical PDAs when sent to home turf with someone inside.
```

### CODE REVIEW

**Comment by jan-antilles** [comment]
AMAZING QoL change, both for doctors and those who end up in the portamed!

**Review by Tarmunora — APPROVED** [review]

**Review by MarkNais — APPROVED** [review]
Code looks good (I did some PDA changes a month ago).
Also, very good QoL change.

File: code/obj/machinery/sleeper.dm [hunk 1]
```diff
@@ -682,6 +682,7 @@
 	mats = 30
 	p_class = 1.2
 	var/homeloc = null
+	var/list/mailgroups = list(MGD_MEDBAY, MGD_MEDRESEACH)
```
Comment (ZeWaka) [inline code/obj/machinery/sleeper.dm:685]: can you document what this list is for?

File: code/obj/machinery/sleeper.dm [hunk 2]
```diff
@@ -744,6 +745,31 @@
 			logTheThing("station", usr, null, "sets [src.name]'s home turf to [log_loc(src.homeloc)].")
 		return
 
+/obj/machinery/sleeper/port_a_medbay/proc/PDA_alert_check() //Yell at doctors to check the thing when it's sent home
```
Comment (ZeWaka) [inline code/obj/machinery/sleeper.dm:748]:
```suggestion
/// Yells at doctors to check the thing when it's sent home
/obj/machinery/sleeper/port_a_medbay/proc/PDA_alert_check()
```

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/machinery/sleeper.dm [hunk 3]
```diff
@@ -682,6 +682,8 @@
 	mats = 30
 	p_class = 1.2
 	var/homeloc = null
+	//Mailgroups it'll try to send PDA notifications to
```
Comment (ZeWaka) [inline code/obj/machinery/sleeper.dm:685]:
```suggestion
	/// Mailgroups it'll try to send PDA notifications to
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**